compiler/rustc_target/src/asm/powerpc.rs

```diff
@@ -48,6 +49,13 @@ impl PowerPCInlineAsmRegClass {
                 }
             }
             Self::freg => types! { _: F32, F64; },
+            Self::vreg => {
+                // FIXME: vsx also supports integers and floats
+                types! {
+                    altivec: VecI8(16), VecI16(8), VecI32(4), VecF32(4);
+                    vsx: VecI64(2), VecF64(2);
+                }
```

Comment:
(Originally I tried to add `I8, I16, I32, I64, F32, F64` to vsx, but I got "rustc-LLVM ERROR: Unknown mismatch in getCopyFromParts!" in the tests passing integers and floats to vreg.)